efficientdet/keras/eval.py

```diff
@@ -26,12 +26,14 @@
 from keras import efficientdet_keras
 from keras import postprocess
 
+import tensorflow as tf
+
 flags.DEFINE_string('val_file_pattern', None,
                     'Glob for eval tfrecords, e.g. coco/val-*.tfrecord.')
 flags.DEFINE_string('val_json_file', None,
                     'Groudtruth file, e.g. annotations/instances_val2017.json.')
 flags.DEFINE_string('model_name', 'efficientdet-d0', 'Model name to use.')
-flags.DEFINE_string('checkpoint', None, 'Location of the checkpoint to run.')
+flags.DEFINE_string('model_dir', None, 'Location of the checkpoint to run.')
 flags.DEFINE_integer('batch_size', 8, 'Batch size.')
```

Comment:
Better to allow both dir and filename as shown in https://github.com/google/automl/commit/ccf8b6ee6261d5f6e2f6f30c247b82bbcf309210